### DISCUSSION TITLE
How to get the cookie using http exec

### DISCUSSION THREAD

**Discussion by KenClemson** [ovh/venom#694] [Q&A]
Hi I cant work out how to get the cookie, is it possible ?

Here is an example of my yaml and Im getting a 200
![image](https://github.com/ovh/venom/assets/48608310/162a5313-6a66-42b2-994c-7f3925e13159)

but I cant see where to find the cookie
![image](https://github.com/ovh/venom/assets/48608310/f8dcd3b2-75b9-41b3-9c1e-bccd19630f75)

please help !

**Comment by yesnault** [comment]
If the executor is missing some options, you can always use a step script with curl to do that.
